packages/ui/src/components/va-data-table/VaDataTable.vue

```diff
@@ -329,6 +333,7 @@ export default defineComponent({
     height: { type: [String, Number] },
     rowBind: { type: null as unknown as PropType<DataTableRowBind> },
     cellBind: { type: null as unknown as PropType<DataTableCellBind> },
+    scrollEventsThreshold: { type: Number },
```

Comment:
```suggestion
    scrollEventsThreshold: { type: Number, default: 0 },
```

Reply:
The main idea is to not fire scroll events if set to `undefined`.

Reply:
Why not? It feels unnatural syncing threshold value (number) with doWeNeedToFireScrollEvent (boolean).

Reply:
done